README.md

```diff
@@ -1,26 +1,50 @@
-
 Node.js
-=====
+=======
 
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/nodejs/node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This repository began as a GitHub fork of
-[joyent/node](https://github.com/joyent/node).
-
-Node.js contributions, releases, and contributorship are under an
+The Node.js project is supported by the
+[Node.js Foundation](https://nodejs.org/en/foundation/). Contributions,
+policies and releases are managed under an
 [open governance model](./GOVERNANCE.md).
-We intend to land, with increasing regularity, releases which are
-compatible with the npm ecosystem that has been built to date for
-Node.js.
+
+## Release Types
+
+The Node.js project maintains multiple types of releases:
+
+* **Stable**: planned and signed releases. Released from active development
+  branches of this repository and versioned by [SemVer](http://semver.org/).
+  Code for Stable releases are organized in this repository by major version
+  number, For example:. [v4.x](https://github.com/nodejs/node/tree/v4.x).
```

Comment:
Maybe go with v5.x rather than v4.x here, just to give people who follow the link the actual current latest Stable.